zetaclient/btc_signer.go

```diff
@@ -108,7 +108,12 @@ func (signer *BTCSigner) SignWithdrawTx(to *btcutil.AddressWitnessPubKeyHash, am
 	remaining := total - amount
 	remainingSats, err := getSatoshis(remaining)
 	if err != nil {
-		return nil, err
+		// TODO: amount validation should be done in zeta core to block invalid cctx
```

Comment:
I think there are different validaty check that we could perform for a cctx:
- addresses validaty
- amount
-  len(outbound) <= 2

Created an issue for it https://github.com/zeta-chain/node/issues/1180